crates/aptos/Cargo.toml

```diff
@@ -22,7 +22,7 @@ futures = "0.3.21"
 hex = "0.4.3"
 itertools = "0.10.3"
 rand = "0.7.3"
-regex = "1.1.5"
+regex = "1.1.6"
```

Comment:
just temporarily made this change to verify that cargo complains about the lockfile being out-of-date now